### ISSUE TITLE
XSD.DATE abstraction management

### ISSUE THREAD

**Issue by ofilangi** [askomics/flaskomics#230]
I use AskOmics 4.2.1
If if defined this asbtraction
```
<http://purl.org/dc/terms/dateSubmitted> a owl:DatatypeProperty;
  rdfs:label "submission";
  rdfs:domain baseClass:StudyML;
  rdfs:range xsd:date .
```
I can not retrieve graphics box to manage Date property.

How Should I have to define "Submission" DatatypeProperty ?

**Comment by mboudet** [comment]
The "Date" entity is implemented starting from release 4.3.0 (or current dev branch).
(Not sure it would work in 4.3.0 in any case, but it will not in 4.2.1)

**Comment by ofilangi** [comment]
The boxe date appears int the [4.3.0](https://github.com/askomics/flaskomics/releases/tag/4.3.0)  !
